src/satipc.c

```diff
@@ -463,7 +464,7 @@ int satipc_open_device(adapter *ad)
 	sip->expect_reply = 0;
 	sip->last_connect = 0;
 	sip->sent_transport = 0;
-	sip->session[0] = 0;
+	sip->sleep = 0;
```

Comment:
This is not an error!
The original code is repeated in the same block:
https://github.com/catalinii/minisatip/blob/681f099c8dd6494e24db65129e5ee44d3b6bc96f/src/satipc.c#L457
https://github.com/catalinii/minisatip/blob/681f099c8dd6494e24db65129e5ee44d3b6bc96f/src/satipc.c#L466